src/features/explore/pages/explore-page.test.tsx

```diff
@@ -16,12 +16,14 @@ import { randomNumberBetween } from '@makerx/ts-dossier'
 import { ellipseId } from '@/utils/ellipse-id'
 import { ellipseAddress } from '@/utils/ellipse-address'
 import { createReadOnlyAtomAndTimestamp, createTimestamp } from '@/features/common/data'
+import { useParams } from 'react-router-dom'
 
 describe('explore-page', () => {
   describe('when no blocks are available', () => {
     const myStore = createStore()
 
     it('no latest blocks are displayed', () => {
+      vi.mocked(useParams).mockReturnValue({ round: '1234' })
```

Comment:
```suggestion
```
AFAIK round isn't used as a param here. I also pushed a small change (as part of merging main into the branch) which removes the need for `vi.mocked(useParams).mockReturnValue({})` in the tests that don't use params.